api/schema/aws_account_resource.go

```diff
@@ -52,6 +52,9 @@ var (
 				"name": resource_schema.StringAttribute{
 					Description: "Display name for the AWS account.",
 					Required:    true,
+					Validators: []validator.String{
+						stringvalidator.LengthAtLeast(3),
+					},
```

Comment:
Remove. Where does this come from? I can't find any such validation on the server side.
```suggestion
```

Reply:
right now if the name isnt specified aws account name is set and we returning something else from whats submitted is throwing error in terraform apply.

state change detected sent "" got "123456789012"

hence added it as required.

Reply:
Why 3 though? This should be 1 then, correct?

Reply:
can be 1. but 3 is what i am usually used to seeing in the name fields. I can make it 1